src/components/course/data/tests/hooks.test.jsx

```diff
@@ -1679,46 +1676,61 @@ describe('useCourseListPrice', () => {
   beforeEach(() => {
     jest.clearAllMocks();
     useCourseRedemptionEligibility.mockReturnValue({ data: { listPrice: mockListPrice } });
-    useCourseMetadata.mockReturnValue(mockListPrice || getCoursePrice(baseCourseMetadataValue));
+    useCourseMetadata.mockReturnValue({ data: mockListPrice || getCoursePrice(baseCourseMetadataValue) });
   });
   it('should return the list price if one exist', () => {
     const { result } = renderHook(
       () => useCourseListPrice(),
       { wrapper: Wrapper },
     );
-    expect(result.current).toEqual(mockListPrice);
+    expect(result.current).toEqual({ data: mockListPrice });
   });
-  it('should not return the list price if one doesnt, first fallback, firstEnrollablePaidSeatPrice', () => {
+  it('should not return the list price if one doesnt exist, first fallback price, fixed_price_usd', () => {
     const updatedListPrice = undefined;
     useCourseRedemptionEligibility.mockReturnValue({ data: { listPrice: updatedListPrice } });
-    useCourseMetadata.mockReturnValue(updatedListPrice || getCoursePrice(baseCourseMetadataValue));
+    useCourseMetadata.mockReturnValue({ data: updatedListPrice || getCoursePrice(baseCourseMetadataValue) });
     const { result } = renderHook(
       () => useCourseListPrice(),
       { wrapper: Wrapper },
     );
-    expect(result.current).toEqual(baseCourseMetadataValue.activeCourseRun.firstEnrollablePaidSeatPrice);
+    expect(result.current).toEqual({ data: [baseCourseMetadataValue.activeCourseRun.fixedPriceUsd] });
   });
-  it('should not return the list price if one doesnt, second fallback, entitlements', () => {
+  it('should not return the list price if one doesnt exist, second fallback price, firstEnrollablePaidSeatPrice', () => {
```

Comment:
nit: I'm still not entirely sure if I'm reading this test case name (and others like it) right. 

Is this essentially saying if a list price doesn't exist, fallback to firstEnrollablePaidSeatPrice?

Reply:
Updated